### ISSUE TITLE
Improvements

### ISSUE THREAD

**Issue by santaclaus21** [pkerspe/ckeditor-bootstrap-collapsibleItem#5]
1) allow to change color of text/glypthicon, background.
2) change glypth icon while open to "^"
3) open/close by click on panel-heading, not only on links.

**Comment by pkerspe** [comment]
HI santaclaus21,

thank you for your input.
As for your first point: why not change/override the color in your local css files? I do not feel like it is the task of the plugin, actually the overall site css should take care of a consistent design / style. Thus allowing the user to set the colors in the editor would contradict the idea of a central styleguide. Or is you intention to use individual colors for each collapsible container? Again then you could use css to overwrite the plugins default color

Point 2 might be something you could also solve via custom css (using transform: rotate() on active collapsibles chevron)

3. this is a point that indeed requires changes to the plugin itself. Will check

**Comment by santaclaus21** [comment]
1) not for me. I am only web developer. There is a woman 💃 and she like to use a lot of colors. Why textcolor button isn't active (?), my ACF allow to use color on span (?). This is a point. Personally I don't like to use more than one color, but it is not exaxtly for me :)
2) maybe you have right. Probably you can not do this in html. 
3) I have done it a few weeks ago, using smarty loop...

```
<div class="container" style="text-align: center;">		
	<div class="panel-group" id="transaction-list">
	{foreach key=tr_k item=tr_i from=$trans_tbl name=trans_tbl }
	{assign var="status" value=$tr_i.status} 
		<div class="panel panel-default ">
				<div class="panel-heading accordion-toggle question-toggle collapsed trans-opener" data-toggle="collapse" data-parent="#transaction-list" data-target="#trans-det-{$tr_i.id}">
					<b>ID:</b> {$tr_i.numerek}&nbsp;&nbsp;
					<b>{t}Data{/t}:</b> {$tr_i.kiedy}&nbsp;&nbsp;
					<b>{t}kwota{/t}:</b> {$tr_i.kwota} {$tr_i.waluta}&nbsp;&nbsp;
					<b>{t}status{/t}:</b> <span style="color: {$stat_col.$status}">{t}{$tr_i.stat_nazwa}{/t}</span>
					<a class="icon-info-sign info-status" data-info="status-legend-{$tr_i.status}" title="" aria-hidden="true"></a>&nbsp;&nbsp;
				</div>
				<div id="trans-det-{$tr_i.id}" class="panel-collapse collapse" style="height: 0px;">
					<div class="panel-body"></div>
				</div>
		</div>
	{/foreach}											
	</div>
</div>	
```

**Comment by santaclaus21** [comment]
re 1) line:30:
```
allowedContent: 'span strong em u span;*{color}'
```

**Comment by pkerspe** [comment]
done and published in version 0.5
